src/redux/store/store.ts

```diff
@@ -1,27 +1,37 @@
 import { applyMiddleware, createStore, Store } from "redux";
-import reduxImmutableStateInvarient from "redux-immutable-state-invariant";
+import reduxImmutableStateInvariant from "redux-immutable-state-invariant";
 import { createLogger } from "redux-logger";
 import thunk from "redux-thunk";
 import rootReducer from "../reducers";
 import { IApplicationState } from "../../models/applicationState";
 import { createLocalStorage, mergeInitialState } from "../middleware/localStorage";
+import { Env } from "../../common/environment";
 
 /**
- * Creates initial redux store from initial aplication state
+ * Creates initial redux store from initial application state
  * @param initialState - Initial state of application
  * @param useLocalStorage - Whether or not to use localStorage middleware
  */
 export default function createReduxStore(
     initialState?: IApplicationState,
     useLocalStorage: boolean = false): Store {
     const paths: string[] = ["appSettings", "connections", "recentProjects"];
-    const middlewares = useLocalStorage
-        ? applyMiddleware(thunk, reduxImmutableStateInvarient(), createLogger(), createLocalStorage({ paths }))
-        : applyMiddleware(thunk, reduxImmutableStateInvarient(), createLogger());
+
+    const middlewares = [];
```

Comment:
You'll need to write a unit test or 2 to ensure we meet the code coverage requirements.